jlm/rvsdg/region.hpp

```diff
@@ -390,6 +396,17 @@ class region
   [[nodiscard]] static size_t
   NumRegions(const jlm::rvsdg::region & region) noexcept;
 
+  /**
+   * Converts \p region and all of its contained structural nodes with subregions to a tree in
+   * ASCII format.
```

Comment:
An example of the output would be nice in this comment

Reply:
I added an example to the documentation of the two methods.